tests/integration/reduce/test_reduce.py

```diff
@@ -195,3 +208,23 @@ def test_uses_before_no_reduce_real_executor_uses():
     # assert no reduce happened
     assert len(resp[0].docs) == 1
     assert resp[0].docs[0].id == 'fake_document'
+
+
+def test_reduce_status():
+    n_shards = 2
+    flow = Flow(port_expose=exposed_port).add(
+        uses=ExecutorStatus, name='pod0', shards=n_shards, polling='all'
+    )
+
+    with flow as f:
+        da = DocumentArray([Document() for _ in range(5)])
+        resp = Client(port=exposed_port).post(
+            '/status', parameters={'foo': 'bar'}, inputs=da, return_results=True
+        )
+
+    assert resp[0].parameters['foo'] == 'bar'
+    assert len(resp[0].parameters['__results__']) == n_shards
+
+    for _, param in resp[0].parameters['__results__'].items():
```

Comment:
can you please assert also on the text of the document being `doc.text = 'exec-status'`?

Reply:
good catch once the test are failing now. However it seems that this bug was here before.